src/base/http/server.cpp

```diff
@@ -108,9 +109,13 @@ void Server::incomingConnection(const qintptr socketDescriptor)
         static_cast<QSslSocket *>(serverSocket)->startServerEncryption();
     }
 
-    auto *c = new Connection(serverSocket, m_requestHandler, this);
```

Comment:
So why not keep this logic?

Reply:
IMO it's cleaner to keep these two classes separated.